src/argvalues.rs

```diff
@@ -118,6 +121,8 @@ impl ArgValues {
             (None, false)
         };
 
+        let force_srgb_off = matches.is_present("force_srgb_off");
```

Comment:
Here and elsewhere (`README.md`, `src/cli.yml`), I'd prefer `force-srgb-off` (i.e. dashes instead of underscores

Reply:
It seems there is another parameter that is making use of underscores `anisotropic_max`. Should this one be changed to dashes to while I'm at it :D

Reply:
Doh! I missed that. In that case, for the sake of both uniformity and backwards compatibility, let's just use underscores for everything :P

Reply:
Reverted :D

Reply:
Thank you!